config/deploy.example.rb

```diff
@@ -59,7 +59,7 @@
     # Generate unique secret token
     run %Q{if [ ! -f #{shared_configs}/secret_token.rb ]; then
```

Comment:
this line is not necessary anymore, as we are removing `secret_token.rb` in this PR.